hackathon_site/hardware/models.py

```diff
@@ -27,7 +27,7 @@ class Meta:
     quantity_available = models.IntegerField(null=False)
     notes = models.TextField(null=True)
     max_per_team = models.IntegerField(null=True)
-    picture = models.FileField(upload_to="hardware/pictures/", null=False)
+    picture = models.ImageField(upload_to="uploads/hardware/pictures/", null=False)
```

Comment:
Note that this will upload to media/uploads/hardware/pictures. I also don't know why we would be calling it uploads since we're not uploading, did you mean to call the resumes one uploads?

Reply:
This is because of what I talked about in the readme and offline, we need a way to distinguish which uploaded files can be served by nginx and which can't (because they require some permissions). In the readme I said that anything under `/media/uploads/` can be served by nginx as public content, anything elsewhere under `/media/` (like `/media/resumes/`) needs to be served custom by a django view (that checks permissions, for example).

> since we're not uploading,

I assumed this actually was going to be done by an upload through the admin site. It doesn't make sense to have it as static files since then we can't add them easily, so even if we have a migration that creates them initially having them under `/media/uploads/` means we can use nginx to serve them per above.